tests/unit/amo/components/TestAddon.js

```diff
@@ -630,18 +631,6 @@ describe(__filename, () => {
     );
   });
 
-  it('sets a title for the description of a lightweight theme', () => {
-    const root = shallowRender({
-      addon: createInternalAddon({
-        ...fakeAddon,
-        type: ADDON_TYPE_THEME,
-      }),
-    });
-    expect(root.find('.AddonDescription').prop('header')).toContain(
-      'About this theme',
-    );
```

Comment:
Do we have such a test case somewhere else?

Reply:
well I broke it up into types... lightweight theme type is under this test name now:

`does not display a lightweight theme's show more card`

Reply:
will that work?

Reply:
oops, I am sorry. I meant "assertion" and not "test case". We need to check the `header` prop once at least.

Reply:
ok if the header is part of the showMore card which is not used in this case, then I am not sure how to check this in a different way than I already have:

```javascript
it("does not display a lightweight theme's show more card", () => {
    const root = shallowRender({
      addon: createInternalAddon({
        ...fakeAddon,
        type: ADDON_TYPE_THEME,
        summary: 'my theme is very cool',
      }),
    });

    expect(root.find('.AddonDescription')).toHaveLength(0);
  });
```

Reply:
mmh what I wanted is an assertion for `header === 'About this theme'` but now I realize there are test cases for that above in the file (not shown in the GitHub diff panel), and we don't want that anyway for lightweight themes.

Nevermind 😄

Reply:
ok cool : )